force-app/main/default/objects/Hierarchy_Settings__c/fields/Lead_Naming_Account_RTypes__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Lead_Naming_Account_RTypes__c</fullName>
```

Comment:
```suggestion
    <fullName>Lead_Converted_Account_RTypes__c</fullName>
```

Reply:
I'm not committing this suggestion directly from here as it requires quite a few other changes, but have made the change.